README.md

```diff
@@ -30,20 +42,46 @@ cargo dsync -i schema.rs -o models
 Now we have everything we need!
 
 ```rust
-use models::todos;
-
 async fn demo(db: Connection) {
+  /*
+    CRUD examples
+  */
   let created_todo = todos::create(&mut db, todos::CreateTodo {
```

Comment:
```suggestion
  let created_todo = Todo::create(&mut db, todos::CreateTodo {
```

unless i missed something, this should likely be the actual case (error from before this PR)